### PR TITLE
exec: remove unused var

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

File: libpod/oci_conmon_linux.go [hunk 1]
```diff
@@ -1213,7 +1213,7 @@ func startCommandGivenSelinux(cmd *exec.Cmd) error {
 
 // moveConmonToCgroupAndSignal gets a container's cgroupParent and moves the conmon process to that cgroup
 // it then signals for conmon to start by sending nonse data down the start fd
-func (r *ConmonOCIRuntime) moveConmonToCgroupAndSignal(ctr *Container, cmd *exec.Cmd, startFd *os.File, uuid string) error {
+func (r *ConmonOCIRuntime) moveConmonToCgroupAndSignal(ctr *Container, cmd *exec.Cmd, startFd *os.File) error {
```
Comment (TomSweeneyRedHat) [inline libpod/oci_conmon_linux.go]: I'm surprised golint or the compiler didn't upchuck over this?

**Comment by mheon** [comment]
LGTM

**Pull request merged**